lib/data/context.js

```diff
@@ -46,6 +48,18 @@ module.exports = function (context) {
   };
 };
 
+function replaceTitle(title, dataRow) {
+  if (typeof dataRow.data === 'object') {
+    return `${title} | ${JSON.stringify(dataRow.data)}`;
```

Comment:
Looks like JSON.stringify should work fine for all cases :)

Reply:
fix this.
And use for all cases